lib/temping.rb

```diff
@@ -53,6 +53,14 @@ def build
       end
     end
 
+    def model_parent_class
+      if ActiveRecord::VERSION::MAJOR > 4
+        ApplicationRecord
```

Comment:
Should this code also check that `ApplicationRecord` is defined?
It's not mandatory to have that in Rails 5, and some apps upgrading from 4 may skip it, causing some unexpected issues.